monitor/service.go

```diff
@@ -451,6 +451,7 @@ func (s *Statistic) ValueNames() []string {
 	return a
 }
 
+// Statistics ....
```

Comment:
This needs a proper comment. Perhaps something like `Statistics is a collection of sortable statistics.`